src/categories.toml

```diff
@@ -297,6 +297,33 @@ formats. Format-specific parsers belong in other, more specific \
 categories.\
 """
 
+[rendering]
+name = "Rendering"
+description = """
+Crates related to rendering 2D or 3D graphics, \
+usually on the screen and with the help of the graphics card.\
```

Comment:
"Online rendering" doesn't exist though. "Real-time rendering" is a really established terminology.